linux/keyman-config/keyman_config/uninstall_kmp.py

```diff
@@ -146,14 +144,18 @@ def uninstall_kmp_user(packageID):
             uninstall_keyboards_from_ibus(keyboards, kbdir)
     else:
         logging.warning("could not uninstall keyboards")
-    delete_dir(kbdir)
-    logging.info("Removed user keyman directory: %s", kbdir)
+    if not os.path.isdir(kbdir):
+        msg = _("Keyboard directory for %s does not exist." % packageID)
```

Comment:
I'm debating whether it is useful to show this error to the user (in a dialog). Since he can't do anything about it, and the directory is already gone it seem we don't need to show it. Thoughts?

Reply:
Don't show it. Just log it. Generally, if there's nothing the user can do about it, then probably better to avoid showing it (otherwise it just causes distress for nothing).